community-plugins.json

```diff
@@ -40,5 +40,13 @@
         "author": "Claude",
         "description": "Concise Summary of Patient's Medical History",
         "prompt": "You will receive a conversation between a healthcare provider and a patient discussing the patient's medical history. Your task is to carefully review the conversation and identify key information about the patient's past and current health conditions, surgeries, medications, allergies, family history, and any other relevant medical details mentioned. Use this information to generate a clear, concise, and well-organized summary of the patient's medical history. The summary should be divided into appropriate sections such as 'Past Medical History', 'Surgical History', 'Medications', 'Allergies', 'Family History', and 'Social History'. Ensure that the summary is written using professional medical terminology and maintains patient confidentiality. If any important aspects of the medical history are not discussed in the conversation, note them as 'Not Mentioned' under the relevant section headings."
-    }
+    },
+    {
+        "id": "ai-companion",
+        "name": "Her",
```

Comment:
This name is a bit of potential copyright issue...

Reply:
fixed it